### ISSUE TITLE
Release new version with changes in it

### ISSUE THREAD

**Issue by PizzaPete** [alepop/stencil-env#14]
Dear maintainer,

The current version (0.3.0) doesn't include the latest changes. My built version for example doesn't take any arguments (options), although this was added in pull request #1 .

Kind regards,
Pieter

**Comment by PizzaPete** [comment]
The built version which is on NPM:
```javascript
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var dotenv = require('dotenv');
exports.env = function () {
    var result = dotenv.config().parsed;
    return {
        name: 'env',
        transform: function (sourceText, id) {
            var code = sourceText;
            Object.keys(result).forEach(function (key) {
                code = code.replace("process.env." + key, "\"" + result[key] + "\"");
            });
            return new Promise(function (resolve) {
                return resolve({
                    id: id,
                    code: code,
                });
            });
        },
    };
};
```

The built version after checking out the repo, npm installing and running: 
> npm run build

```javascript
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var dotenv_1 = require("dotenv");
// https://stackoverflow.com/a/1144788/9238321
function replaceAll(str, find, replace) {
    return str.replace(new RegExp(find.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, "\\$1"), 'g'), replace);
}
function env(options) {
    var result = dotenv_1.config(options).parsed;
    return {
        name: 'env',
        transform: function (sourceText, id) {
            var code = sourceText;
            Object.keys(result).forEach(function (key) {
                code = replaceAll(code, "process.env." + key, "\"" + result[key] + "\"");
            });
            return new Promise(function (resolve) {
                return resolve({
                    id: id,
                    code: code,
                });
            });
        },
    };
}
exports.env = env;
;
```

**Comment by alepop** [comment]
@PizzaPete thank you! I've just released version `0.4.0` with the correct code.
